data/13-georgia/README.md

```diff
@@ -0,0 +1,5 @@
+Sourced from Georgia General Assembly's [Legislative and Congressional Reapportionment Office](http://www.legis.ga.gov/Joint/reapportionment/en-US/default.aspx) on 2017-03-11:
```

Comment:
In the future these should be turned into a JSON file using a standardized format. But until then, Markdown goes a long way.